lib/vm/src/cache.rs

```diff
@@ -16,10 +17,21 @@ use crate::wasm_store::{load, save, wasm_hash};
 static WASM_DIR: &str = "wasm";
 static MODULES_DIR: &str = "modules";
 
+#[derive(Debug, Default, Clone)]
+struct Stats {
+    hits_instance: u32,
+    hits_module: u32,
+    misses: u32,
+}
+
 pub struct CosmCache<S: Storage + 'static, A: Api + 'static> {
     wasm_path: PathBuf,
     modules: FileSystemCache,
-    instances: Option<LruCache<WasmHash, Instance<S, A>>>,
+    instances: Option<LruCache<WasmHash, wasmer_runtime_core::Instance>>,
+    stats: Stats,
```

Comment:
Is this readable from outside?
I think it would be useful to make public readable

Reply:
hmm, I don't know. At the moment it is only used in the same file. Feel free to explore